pkg/scheduler/objects/preemption.go

```diff
@@ -548,14 +548,18 @@ func (p *Preemptor) TryPreemption() (*Allocation, bool) {
 			victimQueue.IncPreemptingResource(victim.GetAllocatedResource())
 			victim.MarkPreempted()
 			log.Log(log.SchedPreemption).Info("Preempting task",
-				zap.String("applicationID", victim.GetApplicationID()),
-				zap.String("allocationKey", victim.GetAllocationKey()),
+				zap.String("victimApplicationID", victim.GetApplicationID()),
+				zap.String("victimAllocationKey", victim.GetAllocationKey()),
 				zap.String("nodeID", victim.GetNodeID()),
-				zap.Stringer("resources", victim.GetAllocatedResource()))
+				zap.Stringer("resources", victim.GetAllocatedResource()),
+				zap.String("askApplicationID", p.ask.applicationID),
+				zap.String("askAllocationKey", p.ask.allocationKey),
+				zap.String("askQueue", p.ask.String()),
+				zap.String("victimQueue", victimQueue.Name))
```

Comment:
Main comment: sort the info you log.
* start with requesting details: ask
* add requesting resource details 
* keep all victim info and requesting info grouped together
* use same order in victim and requesting detail: app, key, queue, resource